importer.py

```diff
@@ -43,11 +43,12 @@ def wrapper(*args, **kwargs):
             try:
                 return func(*args, **kwargs)
             except SpotifyException as exception:
-                if exception.http_status != 429:
+                if exception.http_status not in [429, 404]:
                     raise exception
 
                 if 'retry-after' in exception.headers:
                     sleep(int(exception.headers['retry-after']) + 1)
+                    
             except ReadTimeout as exception:
```

Comment:
The body of this exception handling is the same as below. Let's have only one with `Exception as exception`